lib/tasks/migrate.rake

```diff
@@ -78,5 +78,25 @@ namespace :migrate do
       end
       puts "Updated #{ missing_token_count } users have unsubscribe tokens."
     end
+
+    desc "Create project preferences for projects classified on"
+    task create_project_preferences: :environment do
+      project = Project.find(ENV["PROJECT_ID"])
+
+      if user = User.find_by(id: ENV["USER_ID"])
+        p "Updating: #{user.login}"
+        UserProjectPreference.create!(user: user, project: project)
```

Comment:
this will throw an error if the user / project preference already exists. do you want to handle it?